dashboard/test/ui/features/containedLevels.feature

```diff
@@ -0,0 +1,51 @@
+@as_taught_student
+@eyes
+Feature: Contained Levels
+
+Scenario: Applab with free response contained level
+  When I open my eyes to test "applab contained level"
+  Given I am on "http://learn.code.org/s/allthethings/stage/18/puzzle/15"
+  And I rotate to landscape
+  And I wait to see "#runButton"
+  Then I see no difference for "initial load"
+  Then I press keys "This is my answer" for element ".response"
+  And I see no difference for "answer entered"
+  Then I press "runButton"
+  And I see no difference for "level run"
+  # At this point, we should have submitted our result to the server, do
+  # a reload and make sure we have the submission
+  Then I am on "http://learn.code.org/s/allthethings/stage/18/puzzle/15"
+  And I rotate to landscape
+  And I wait to see "#runButton"
+  And I see no difference for "reloaded with contained level answered"
+  Then I press "runButton"
+  And I press "finishButton"
+  And I see no difference for "finished level with contained level"
+  And I press "continue-button"
+  # Make sure continue takes us to next level
+  And I wait until current URL contains "/stage/19/"
+  Then I close my eyes
+
+Scenario: Gamelab with multiple choice contained level
+  When I open my eyes to test "gamelab contained level"
+  Given I am on "http://learn.code.org/s/allthehiddenthings/stage/2/puzzle/1"
+  And I rotate to landscape
+  And I wait to see "#runButton"
```

Comment:
Could replace these three lines with [`Given I am on the 1st GameLab test level`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/step_definitions/gamelab.rb#L15)